sktime/clustering/base/base.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+"""Base classes for clustering."""
+
+__author__ = ["Christopher Holder", "Tony Bagnall"]
+__all__ = [
+    "BaseCluster",
+    "ClusterMixin",
+    "BaseClusterCenterInitializer",
+    "BaseClusterAverage",
+    "BaseApproximate",
+    "DataFrame",
+    "NumpyArray",
+    "NumpyOrDF",
+    "NumpyRandomState",
+    "CenterCalculatorFunc",
+]
+
+import pandas as pd
+import numpy as np
+from sktime.base import BaseEstimator
+from sktime.utils.data_processing import from_nested_to_2d_array
+from typing import Union, Callable
+
+DataFrame = pd.DataFrame
+NumpyArray = np.ndarray
+NumpyOrDF = Union[DataFrame, NumpyArray]
+NumpyRandomState = Union[np.random.RandomState, int]
+CenterCalculatorFunc = Callable[[NumpyArray], NumpyArray]
+
+
+class BaseCluster(BaseEstimator):
+    """Base Clusterer"""
+
+    def __init__(self):
+        super(BaseCluster, self).__init__()
+
+    def fit(self, X: NumpyOrDF) -> None:
+        """
+        Method that is used to fit the clustering algorithm
+        on the dataset X
+
+        Parameters
+        ----------
+        X: Numpy array or Dataframe
+            sktime data_frame or numpy array to train the model on
+
+        Returns
+        -------
+        self
+            Fitted estimator
+        """
+        self._is_fitted = False
```

Comment:
This isn't strictly necessary here as it's set in the constructor of the `BaseEstimator`
```suggestion
```

Reply:
This must have got readded by mistake when i was fixing whatever happened with the pr the other day good spot